rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.7.3-release-notes.adoc

```diff
@@ -0,0 +1,13 @@
+= RPA Builder 6.7.3 Release Notes
+
+RPA Builder is the on-premise application that enables you to build automations by specifying the action steps of the process.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
```

Comment:
date?

Reply:
Not included because this is just an internal release.